Makefile

```diff
@@ -0,0 +1,51 @@
+# Variables
+
+target_container ?= php
+php_sources ?= .
+
+# Utils
+
+vendor: composer.json composer.lock
+	make -C $$PWD composer-install
+
+.PHONY: composer-add-github-token
+composer-add-github-token:
+	docker-compose run --rm $(target_container) composer config --global github-oauth.github.com $(token)
+
+.PHONY: composer-update
```

Comment:
.PHONY: composer-update
composer-update:
	docker-compose run --rm $(target_container) php -d memory_limit=-1 /usr/local/bin/composer update $(options)

Reply:
Fixed in 27d41f822f32d2b15ce1dfd1f214c1e81c5ee876